app/lib/three_scale/invitation_email_delivery_observer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ThreeScale::InvitationEmailDeliveryObserver
```

Comment:
I think that @hallelujah used to tell to Cadu to not put things called from the initializers in `app/lib` and instead put them in `lib`. But @hallelujah should be the person saying where this class should go 😄